examples/index.html

```diff
@@ -262,129 +262,128 @@ <h2 class="bg-dark">Usage</h2>
                         </div>
                     </a>
                 </div>
+            </div>
+        </section>
 
+        <section class="col-12 mt-2 mb-2">
+            <a id="style" class="anchor-from-summary"></a>
+            <h2 class="bg-dark">Style</h2>
+            <div class="columns">
                 <div class="col-4 p-2">
-                    <a href="./diagram-navigation/index.html" target="_blank" class="card-link">
+                    <a href="style/custom-colors/index.html" target="_blank" class="card-link">
                         <div class="card p-2">
                             <div class="card-header">
-                                <div class="card-title">Diagram Navigation</div>
+                                <div class="card-title">Custom colors</div>
                                 <div class="card-subtitle"></div>
                             </div>
                             <div class="card-image">
-                                <img src="static/img/preview_diagram_navigation.png" class="img-responsive" alt="navigate within the diagram">
+                                <img src="static/img/preview/style/preview_custom_colors.png" class="img-responsive" alt="custom colors">
```

Comment:
:question: could we remove the `preview` prefix from all file names? the files are now in a preview subfolder so the prefix is useless and make the name more complex for nothing

Reply:
Yes, I said myself the same thing ^^
But, I wanted to be homogeneous with the rest  😉 

So, if we agree, I'll change it.